presto-docs/src/main/sphinx/release/release-324.rst

```diff
@@ -0,0 +1,44 @@
+===========
+Release 324
+===========
+
+General Changes
+---------------
+
+* Fix query failure when ``CASE`` operands have different types. (:issue:`1825`)
+* Add support for ``ESCAPE`` clause in ``SHOW CATALOGS LIKE ...``. (:issue:`1691`)
+* Allow references to tables in the enclosing query when using ``.*``. (:issue:`1867`)
+* Configuration properties for iterative optimizer no longer have ``experimental.`` prefix. (:issue:`1875`)
+* Configuration properties for cost-based optimizer no longer have ``experimental.`` prefix. (:issue:`1875`)
+* Configuration properties for spill no longer have ``experimental.`` prefix. (:issue:`1875`)
+
+Security Changes
+----------------
+
+* Adding access control check when accessing ``SHOW CREATE TABLE`` and
```

Comment:
This needs a better explanation (and the grammar is wrong)

Reply:
Any suggestions? What about something like "Perform access control checks when displaying table or view definitions with ``SHOW CREATE``"